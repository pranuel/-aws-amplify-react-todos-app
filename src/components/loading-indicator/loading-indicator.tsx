import React from "react";
import "./loading-indicator.css";

export const LoadingIndicator = () => (
  <div className="lds-ripple">
    <div/>
    <div/>
  </div>
);
