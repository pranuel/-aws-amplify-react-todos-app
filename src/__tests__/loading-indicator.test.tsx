import React from "react";
import renderer from "react-test-renderer";
import { LoadingIndicator } from "../components/loading-indicator/loading-indicator";

describe("LoadingIndicator", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<LoadingIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
