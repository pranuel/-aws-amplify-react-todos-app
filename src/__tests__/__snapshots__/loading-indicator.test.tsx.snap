// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator should render correctly 1`] = `
<div
  className="lds-ripple"
>
  <div />
  <div />
</div>
`;
